html/pfappserver/root/src/components/new/BaseFormGroupToggle.vue

```diff
@@ -18,11 +18,8 @@
     :invalid-feedback="inputInvalidFeedback"
     :valid-feedback="inputValidFeedback"
   >
-    <base-input-range
+    <base-input-toggle
```

Comment:
3-state toggles have regressed to 2-states only.

Before:

![Screenshot 2023-06-08 at 9 43 19 AM](https://github.com/inverse-inc/packetfence/assets/3904468/e417216e-6599-4d2d-ad7f-482a3bf86163)


After:

![Screenshot 2023-06-08 at 9 42 20 AM](https://github.com/inverse-inc/packetfence/assets/3904468/9f2b50ea-a571-4084-ad73-7ed78477b5b5)